lib/IAMClient.js

```diff
@@ -206,6 +206,7 @@ class IAMClient {
             signatureFromRequest: signature,
             hashAlgorithm: algo,
             accessKey,
+            requestContext: options.requestContext,
```

Comment:
Soo I'm guessing this is directly provided by the "user" of vaultclient ?
What should this requestContext contain ? Could the documentation be updated to describe that ?